### PR TITLE
LiveSync support

### PR DESCRIPTION
**Pull request by Fatme**
https://github.com/NativeScript/nativescript-cli/issues/523

### CODE REVIEW

**Comment by Fatme** [comment]
@fealebenpae @jasssonpet

File: build/project-template/__PROJECT_NAME__/main.m [hunk 1]
```diff
@@ -16,15 +16,27 @@
...
-
+        NSString *applicationPath = [[NSBundle mainBundle] bundlePath];
+        
+        NSString *libraryPath = [NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES) firstObject];
+        NSString *liveSyncPath = [NSString pathWithComponents:@[libraryPath, @"Application Support", @"LiveSync"]];
+        NSString *appFolderPath = [NSString pathWithComponents:@[liveSyncPath, @"app"]];
+        
+        NSArray *appDocumentsContents = [[NSFileManager defaultManager] directoryContentsAtPath:appFolderPath];
+        if(appDocumentsContents.count > 0)
+        {
```
Comment (fealebenpae) [inline build/project-template/__PROJECT_NAME__/main.m]: We put opening curly brackets on the same line as the preceding expression.

**Comment by fealebenpae** [comment]
:+1:, save for a styling nitpick.

**Comment by jasssonpet** [comment]
run ci

**Pull request merged**